optional_plugins/resultsdb/avocado_resultsdb/__init__.py

```diff
@@ -198,3 +209,11 @@ def run(self, args):
                                                 default=None)
             if resultsdb_logs is not None:
                 args.resultsdb_logs = resultsdb_logs
+
+        resultsdb_note_limit = getattr(args, 'resultsdb_note_limit', None)
+        if resultsdb_note_limit is None:
+            resultsdb_note_limit = settings.get_value('plugins.resultsdb',
+                                                      'note_size_limit',
+                                                      default=None)
+            if resultsdb_note_limit is not None:
+                args.resultsdb_note_limit = resultsdb_note_limit
```

Comment:
This commit looks good. Could you please send it together with the first commit (`resultsdb: Show result log url in human mode`) in a separate PR?